src/Sulu/Bundle/AdminBundle/Resources/translations/admin.de.json

```
@@ -238,5 +238,19 @@
     "sulu_admin.%count%_blocks_duplicated": "{count} {count, plural, =1 {Block} other {Blöcke}} dupliziert.",
     "sulu_admin.%count%_blocks_removed": "{count} {count, plural, =1 {Block} other {Blöcke}} gelöscht.",
     "sulu_admin.%count%_blocks_cut": "{count} {count, plural, =1 {Block} other {Blöcke}} zur Zwischenablage ausgeschnitten.",
-    "sulu_admin.%count%_blocks_pasted": "{count} {count, plural, =1 {Block} other {Blöcke}} eingefügt."
+    "sulu_admin.%count%_blocks_pasted": "{count} {count, plural, =1 {Block} other {Blöcke}} eingefügt.",
+    "sulu_admin.writing_assistant_title": "Schreibassistent",
+    "sulu_admin.writing_assistant_insert": "Einsetzen",
+    "sulu_admin.writing_assistant_prompt_placeholder": "Wie kann sulu dir helfen?",
+    "sulu_admin.writing_assistant_copied_to_clipboard": "Erfolgreich in die Zwischenablage kopiert",
+    "sulu_admin.writing_assistant_initial_message": "Ausgewählter Text",
+    "sulu_admin.writing_assistant_predefined_prompts": "Vordefinierte Prompts",
+    "sulu_admin.writing_assistant_send": "Senden",
+    "sulu_admin.translator_title": "Übersetzer",
+    "sulu_admin.translator_insert": "Einsetzen",
+    "sulu_admin.translator_detected": "Erkannt",
```

Keep admin translations general:

```suggestion
    "sulu_admin.translator": "Übersetzer",
    "sulu_admin.insert": "Einsetzen",
    "sulu_admin.detected": "Erkannt",
```